pkg/model/hetznermodel/loadbalancer.go

```diff
@@ -65,7 +65,11 @@ func (b *LoadBalancerModelBuilder) Build(c *fi.CloudupModelBuilderContext) error
 			hetzner.TagKubernetesClusterName: b.ClusterName(),
 		},
 
-		WellKnownServices: []wellknownservices.WellKnownService{wellknownservices.KubeAPIServer, wellknownservices.KopsController},
+		WellKnownServices: []wellknownservices.WellKnownService{
+			wellknownservices.KopsController,
```

Comment:
Could we rename this to make it easy to understand that it will be always internal?
```suggestion
			wellknownservices.KopsControllerInternal,
```

Reply:
Sure!  I do want to check if it (will be) publicly exposed on any clouds though, as that might be more confusing!

Reply:
It probably will be exposed on some clouds so we may need to add that too and use it when required.